sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -293,7 +293,7 @@ describe("ManagedIdentityCredential", function () {
 
   it("doesn't try IMDS endpoint again once it can't be detected", async function () {
     const credential = new ManagedIdentityCredential("errclient");
-    const DEFAULT_CLIENT_MAX_RETRY_COUNT = 10;
+    const DEFAULT_CLIENT_MAX_RETRY_COUNT = 3;
```

Comment:
aren't you already doing this change in the other PR #20406 ? @sadasant

Reply:
wait, I think I mixed PRs 😅